src/vendor/bullet.js

```diff
@@ -19,11 +20,14 @@ export default function bullet () {
         markerz = markers.call(this, d, i).slice().sort(d3.descending),
         measurez = measures.call(this, d, i).slice().sort(d3.descending),
         measureHeightz = measureHeight.call(this, d, i).slice().sort(d3.descending),
-        rangeMax = getRangeMax.call(this, d, i).slice().sort(d3.descending),
+        rangeMax = rangeMaxx.call(this, d, i).slice().sort(d3.descending),
         g = d3.select(this);
 
+      if (isNaN(rangez[0]) || isNaN(markerz[0]) || isNaN(rangez[0]) ){
+        return;
+      }
       //set the x-axis scale based on the menu setting for universal vs independent
-      if (rangeMax==0) {
+      if (rangeMax[0]==0) {
```

Comment:
⚫️ 🔮 😱 
if you could extract it to a variable it would mean to me a lot, but it's completely up to you